frontend/components/ItemViewer.vue

```diff
@@ -117,13 +111,22 @@ export default {
   async mounted () {
     if (this.id) {
       try {
-        await this.$wikibase.getEntity(this.id, this.$i18n.locale).then((entity) => {
-          this.item = entity
-          this.label = this.$wikibase.getValueByLang(this.item.labels, this.$i18n.locale)
-          this.claimsOrdered = this.orderClaims(this.item.claims)
-          this.showItem = true
-          this.$store.commit('breadcrumb/setItems', this.getBreadcrumbItems(entity))
-        })
+        await this.$wikibase
+          .getEntity(this.id, this.$i18n.locale)
+          .then((entity) => {
+            this.item = entity
+            this.label = this.$wikibase.getValueByLang(
+              this.item.labels,
+              this.$i18n.locale
+            )
+            this.claimsOrdered = this.orderClaims(this.item.claims)
+            console.log('claims ordered', this.claimsOrdered)
```

Comment:
In general we should avoid commit _console.log_ in code unless there is an important reason